site/src/site/releasenotes/groovy-2.6.adoc

```diff
@@ -14,7 +14,7 @@ the new Parrot parser (when enabled) but these versions are not feature complete
 We have left the alpha versions available to help people wanting to port towards Groovy 3.0
 but who are stuck on JDK7. However, we strongly encourage all users on JDK8 or above
 to move directly to Groovy 3.0.
-You should consult the link:../groovy-3.0.html[Groovy 3.0 release notes] for a list of supported language changes.
+You should consult the link:http://groovy-lang.org/releasenotes/groovy-3.0.html[Groovy 3.0 release notes] for a list of supported language changes.
```

Comment:
How about the middle ground: 
`link:/releasenotes/groovy-3.0.html` 
this will accommodate running on localhost as well?

Reply:
The other examples I found are using https://github.com/apache/groovy-website/blob/9af4b84dfa6fb211dcc13637475db1bbf029d74e/site/src/site/releasenotes/groovy-4.0.adoc#L38

Which is why I did it - I'm good with changing my version though
Commit incoming